glide.lock

```diff
@@ -1,215 +1,16 @@
-hash: 9589cf38139f18ca918955308856f05f0f5196ca6189cd6961ffd1aff19d8ba6
```

Comment:
btw is the README /wrt the requirements 'golang 1.4' and 'godeps' still accurate ?